apps/src/templates/sectionsRefresh/QuickAssignTable.jsx

```diff
@@ -1,8 +1,39 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import moduleStyles from './sections-refresh.module.scss';
 
 export default function QuickAssignTable({marketingAudience, courseOfferings}) {
-  return <div>{JSON.stringify(courseOfferings[marketingAudience])}</div>;
+  const TABLE_COUNT = Object.keys(courseOfferings[marketingAudience]).length;
+
+  const renderTable = headerIndex => {
+    return (
+      <table className={moduleStyles.table}>
+        <thead>
+          <tr className={moduleStyles.headerRow}>
+            <td className={moduleStyles.headerCell}>
+              <div>
+                {Object.keys(courseOfferings[marketingAudience])[headerIndex]}
+              </div>
+            </td>
+          </tr>
+        </thead>
+        <tbody>
+          {JSON.stringify(
+            Object.values(courseOfferings[marketingAudience])[headerIndex]
+          )}
+        </tbody>
+      </table>
+    );
+  };
+
+  return (
+    <div className={moduleStyles.multiTables}>
+      {0 < TABLE_COUNT && renderTable(0)}
+      {1 < TABLE_COUNT && renderTable(1)}
+      {2 < TABLE_COUNT && renderTable(2)}
+      {3 < TABLE_COUNT && renderTable(3)}
```

Comment:
I'm not sure if this last line is necessary now that I hear it would be way too cramped with four columns. Should I remove this and just keep 0-2 (count: 3)?

Reply:
Update: edited approach to match the suggested below